bindings/python/src/error.rs

```diff
@@ -7,6 +7,7 @@ use tokenizers::tokenizer::Result;
 #[derive(Debug)]
 pub struct PyError(pub String);
 impl PyError {
+    #[allow(dead_code)]
```

Comment:
That's odd, no ?

Reply:
We need to get rid of `PyError` and `ToPyResult` altogether, but this is for another PR.